lib/MusicBrainz/Server/Data/Search.pm

```diff
@@ -85,13 +85,14 @@ sub search
     my $deleted_entity = undef;
 
     my @where_args;
+    my $extra_columns = '';
 
     if ($type eq 'artist') {
-
         my $where_deleted = 'WHERE entity.id != ?';
         $deleted_entity = $DARTIST_ID;
 
-        my $extra_columns = 'entity.gender, entity.area, entity.begin_area, entity.end_area,' if $type eq 'artist';
+        $extra_columns = 'entity.gender, entity.area,
+                          entity.begin_area, entity.end_area,';
```

Comment:
My only problem with this is that it adds random indentation and whitespace to the string which can show up in query logs and make things harder to read.  Maybe concatenate the string across two lines instead?

Reply:
Does that still happen with a heredoc? https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitImplicitNewlines recommends concatenation over split lines and heredocs over concatenation, but if the heredoc is problematic on query logs then we can just concatenate.

Reply:
I changed this to use heredocs since AFAICT that should be an improvement over the multiline stuff that the file was already using in most places.